Example/Prelude.playground/Pages/reducers.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,66 @@
+//
+// This source file is part of Prelude, an open source project by Wayfair
+//
+// Copyright (c) 2018 Wayfair, LLC.
+// Licensed under the 2-Clause BSD License
+//
+// See LICENSE.md for license information
+//
+
+import Prelude
+
+//
+// Reducers
+//
+
+struct Person { var firstName, lastName: String }
+
+// reducers have an `inout` “state” by default
+let reducer: Reducer<[String], String> = .init { arr, item in arr.append(item) }
+
+// but can also be written with an immutable state
+let reduceCaps: Reducer<[String], String> = .nextPartialResult { arr, item in
+    return arr + [item.uppercased()]
+}
+
+// combine reducers with `<>` to execute them in sequence
+let bigReducer = reducer <> reduceCaps
+["foo", "bar", "baz"].reduce([], bigReducer)
+
+// use `contramap` to make it so existing reducers can chomp other types of values
+let people = [
+    Person(firstName: "foo", lastName: "bar"),
+    Person(firstName: "baz", lastName: "qux")
+]
+let reduceFirstNames = bigReducer.pullback { (person: Person) in person.firstName }
+people.reduce(into: [], reduceFirstNames)
+
+//
+// Use both together!
```

Comment:
This comment now no longer makes a ton of sense (`Changeable` is now on a different playground page)